modules/03 Front End Development/Assignments/08/answer8.html

```diff
@@ -1,44 +1,107 @@
+			<h2 id="achnor_baked_squash">Baked squash</h2>
+
+			<ul>
+				<h3>Ingridients:</h3>
+				<li>1 butternut squash , (1.2kg)</li>
+				<li>olive oil</li>
+				<li>1 red onion</li>
+				<li>1 clove of garlic</li>
+				<li>1 bunch of fresh sage , (30g)</li>
+				<li>10 sun-dried tomatoes</li>
+				<li>75 g vac-packed chestnuts</li>
+				<li>75 g basmati rice</li>
+				<li>75 g dried cranberries</li>
+				<li>1 pinch of ground allspice</li>
+				<li>red wine</li>
+			</ul>
+			<p>Time of cooking: 2H 30M</p>
+			<img alt="lol" src="https://banner2.kisspng.com/20180614/jfq/kisspng-customer-review-5-star-stock-photography-5-stars-5b229bc8d07022.4923293115289947608538.jpg"
+			 style="width: 40%; height: 40%" />
 		</li>
-	</ol>
-	
-	<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
-	<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
-	
-
-	<ul id="Creamy">Creamy Pasta Toss with Smoked Sausage
-		<li>1 (8 ounce) package uncooked pappardelle pasta</li>
-		<li>1 tablespoon olive oil</li>
-		<li>1 (12 ounce) package Hillshire Farm® Rope Smoked Sausage, cut into 1/2" </li>
-		<li>1 (8 ounce) package sliced cremini mushrooms</li>
-		<h5>Time of cooking : 30 m</h5>
-		<img src="https://www.butterflyfields.com/wp-content/uploads/2016/02/4-star-rating.png" alt="4 stars rating" width="200" />
-	</ul>
 
-	<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
-	<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
-	
-	
-	<ul id="Meatloaf">Easy Meatloaf
-		<li>1 1/2 pounds ground beef</li>
-		<li>1 egg</li>
-		<li>1 onion, chopped</li>
-		<li>1 cup dried bread crumbs</li>
-		<h5>Time of cooking : 1 h 10 m</h5>
-		<img src="https://www.garage-gyms.com/wp-content/uploads/2016/02/3-star-rating.png" alt="3 stars rating" width="200" >
+		<li>
+			<h2 id="achnor_cucumber_pickle">Spicy cucumber pickle</h2>
+			<ul>
+				<h3>Ingridients:</h3>
+				<li>6 pickling cucumbers , or 2 regular-sized cucumbers</li>
+				<li>2 banana shallots</li>
+				<li>2 teaspoons mustard seeds</li>
+				<li>½ teaspoon ground turmeric</li>
+				<li>2 star anise</li>
+				<li>75 g caster sugar</li>
+				<li>150 ml vinegar</li>
+			</ul>
+			<p>Time of cooking: 1H 10M</p>
+			<img src="https://banner2.kisspng.com/20180614/jfq/kisspng-customer-review-5-star-stock-photography-5-stars-5b229bc8d07022.4923293115289947608538.jpg"
+			 style="width: 40%; height: 40%" />
+		</li>
 	</ul>
-	
-	<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
-	<a href="https://www.hazi-hinam.co.il/" target="_blank">supermarket - click here</a>
+
+	<h2><a href="https://www.walmart.com/" target="_blank">Supermarket</a></h2>
 </body>
+
 </html>
```

Comment:
correct but the assignment is "modules/03 Front End Development/Assignments/08/answer8.html"